src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,76 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { Link } from 'react-router-dom';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+  itemsOnPage: string[];
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+  itemsOnPage,
+}) => {
+  return (
+    <ul className="pagination">
+      <li className={cn('page-item', { disabled: currentPage === 1 })}>
+        <Link
+          data-cy="prevLink"
+          className={cn('page-link')}
+          aria-disabled={currentPage === 1}
+          to={`?page=${currentPage - 1}&perPage=${perPage}`}
+          onClick={() => currentPage !== 1 && onPageChange(currentPage - 1)}
```

Comment:
The 'onClick' handler for the 'previous' link performs a check inside the handler function. This could be simplified by using the 'preventDefault' method of the event object if 'currentPage' is 1, to prevent the default link behavior.